### PR TITLE
Remove Warning if core file not found

### PR DESCRIPTION
**Pull request by Anu1601CS**
Pull Request for Issue https://github.com/joomla-projects/gsoc18_override_management/pull/9#issuecomment-394153776

### Summary of Changes
Warning Removed

### CODE REVIEW

File: administrator/components/com_templates/Model/TemplateModel.php [hunk 1]
```diff
@@ -310,11 +308,7 @@ private function validateDate($date)
 	private function findPath($paths, $file)
 	{
 		$app = Factory::getApplication();
-
-		if (!$path = Path::find($paths, $file))
-		{
-			$app->enqueueMessage(Text::_('COM_TEMPLATES_ERROR_CORE_FILE_NOT_FOUND'), 'warning');
-		}
+		$path = Path::find($paths, $file);
```
Comment (laoneo) [inline administrator/components/com_templates/Model/TemplateModel.php]: We can just return the path without creating a variable.

**Comment by Anu1601CS** [comment]
@laoneo Done!

**Comment by laoneo** [comment]
Thanks, the pr is fine for me.

File: administrator/components/com_templates/Model/TemplateModel.php [hunk 2]
```diff
@@ -309,14 +307,7 @@ private function validateDate($date)
...
 	{
-		$app = Factory::getApplication();
-
-		if (!$path = Path::find($paths, $file))
-		{
-			$app->enqueueMessage(Text::_('COM_TEMPLATES_ERROR_CORE_FILE_NOT_FOUND'), 'warning');
-		}
-
-		return $path;
+		return Path::find($paths, $file);
```
Comment (astridx) [inline administrator/components/com_templates/Model/TemplateModel.php]: Do we really need our own method for this one line?

Comment (laoneo) [reply administrator/components/com_templates/Model/TemplateModel.php]: Agree

Comment (Anu1601CS) [reply administrator/components/com_templates/Model/TemplateModel.php]: I also agree. I fix this.

**Comment by Anu1601CS** [comment]
@laoneo @astridx Done!

**Pull request merged**